org.eclipse.lsp4e/META-INF/MANIFEST.MF

```diff
@@ -29,8 +29,8 @@ Require-Bundle: org.eclipse.core.runtime;bundle-version="3.12.0",
  org.eclipse.ui.navigator;bundle-version="3.6.100",
  org.eclipse.lsp4j;bundle-version="[0.18.0,0.19.0)",
  org.eclipse.lsp4j.jsonrpc;bundle-version="[0.18.0,0.19.0)",
- org.eclipse.mylyn.wikitext;bundle-version="3.0.40",
- org.eclipse.mylyn.wikitext.markdown;bundle-version="3.0.40",
+ org.eclipse.mylyn.wikitext;bundle-version="[3.0.40,3.0.45]",
```

Comment:
Why are you setting and upper bound. "3.0.40" actually means ">= 3.0.40" according to https://docs.osgi.org/specification/osgi.core/8.0.0/framework.module.html#i3189032 , so this is correct.